app/pdfs/donation_pdf.rb

```diff
@@ -0,0 +1,196 @@
+# Configures a Prawn PDF template for generating Donation receipts
+class DonationPdf
+  include Prawn::View
+  include ItemsHelper
+
+  class DonorInfo
+    attr_reader :name, :address, :email
+
+    def initialize(donation = nil)
+      if donation.nil?
+        @name = @address = @email = nil
+        return
```

Comment:
Is there any valid use case for being passed a nil donation? We should make this a required parameter and probably raise an error if the donation is nil.